docker-compose.yml

```diff
@@ -1,52 +1,65 @@
-version: '3.4'
+version: '2.4'
+
+x-app-service-template: &app
+  image: anycable-demo:0.6.0
+  build:
+    context: .
+    dockerfile: .dockerdev/Dockerfile
+    args:
+      RUBY_VERSION: "2.5.7"
+      NODE_MAJOR: '12'
+      BUNDLER_VERSION: '1.17.1'
+      HIVEMIND_VERSION: '1.0.6'
+  tmpfs:
+    - /tmp
+  environment: &env
+    NODE_ENV: development
+    RAILS_ENV: ${RAILS_ENV:-development}
+
+x-backend: &backend
+  <<: *app
+  volumes:
+    - .:/app:cached
+    - rails_cache:/app/tmp/cache
+    - bundle:/usr/local/bundle
+  tmpfs:
+    - /app/tmp/pids
+  environment:
+    <<: *env
+    REDIS_URL: redis://redis:6379/
+    WEB_CONCURRENCY: 1
+    HISTFILE: /app/tmp/cache/.bash_history
+    EDITOR: vi
+    CABLE_URL: ws://localhost:8080/cable
+  stdin_open: true
+  tty: true
+  depends_on:
+    redis:
+      condition: service_healthy
 
 services:
-  app: &app
-    build:
-      context: .
-      dockerfile: ./Dockerfile
-    image: anycable-demo:0.6.0
-    volumes:
-      - .:/app:cached
-    tmpfs:
-      - /tmp
-  backend: &backend
-    <<: *app
-    volumes:
-      - .:/app:cached
-      - assets:/app/public/assets
-    environment:
-      - RAILS_ENV=${RAILS_ENV:-development}
-      - REDIS_URL=redis://redis:6379/0
-      - CABLE_URL=ws://localhost:8080/cable
-      - ADAPTER=any_cable
```

Comment:
This param is missing in the new config (and that's what likely cause the exception in RPC service).

Reply:
Added and yes, it fixed the issue